### DISCUSSION TITLE
Mega 2560 Pro Won't Turn On?

### DISCUSSION THREAD

**Discussion by LNRC** [sanni/cartreader#363] [Need help]
I'm sorry if this question is already answered somewhere - I've been banging my head, looking to figure this out for hours now.

I'm trying to work with a battery free build of the Sanni Cart Reader, and I am having a lot of trouble getting the mega 2560 pro to turn on when plugged into a computer. It lights up when I touch a multimeter current to the dupont cable joint (where the SMD cable was removed), and it has even powered on enough from the multimeter to be detected on my computer for a few seconds. However, when I plug it in on its own, nothing happens. When I plug it into the wall, nothing happens. I got a USB A - USB B Micro cable specifically for arduino, and I'm not having much luck. I'm not sure if my laptop has too little power to activate the arduino, or if something went wrong when I removed the SMD fuse.  I'm pretty new to working with electronics so I wouldn't be surprised if I'm missing something really basic. Unfortunately, most of the information online covers the non-pro Mega 2560.

Any help is greatly appreciated. Thank you!

**Comment by sanni** [comment]
Once the SMD fuse has been removed the Mega 2560 Pro needs to be plugged into the main PCB and a cart slot adapter PCB needs to be attached to the main PCB too and the switch, either on the battery module or the one you soldered onto the main PCB, needs to be turned on for the Mega 2560 Pro to turn on. The Mega 2560 Pro will not work anymore on its own.

**Reply by LNRC** [reply]
Ah, thank you! I need to make sure I didn't leave a loose connection anywhere then.

**Reply by LNRC** [reply]
I want to follow up with an additional thanks for responding to this question when I was missing a pretty obvious detail (the embed pro has male pins for a reason, you need to put it INTO the female pins to work). Your fast response, even to simple questions like these, is really amazing. Thank you for building this project and taking such great care to maintain it! 

> Once the SMD fuse has been removed the Mega 2560 Pro needs to be plugged into the main PCB and a cart slot adapter PCB needs to be

**Reply by orion-alathorn** [reply]
2 years later you are still helping with this response lol, prepped my mega 2560 pro by removing the fuse and then went to program it and couldn't figure out why it wasn't working and thought I had a dud board, turns out I just had a brain fart and your reply helped me realize what I did, thank you very much kind sir
